src/main/java/seedu/address/model/task/Title.java

```diff
@@ -4,56 +4,56 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Task's name in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ * Represents a Task's title in the address book.
```

Comment:
Do you wanna change "address book" to "PlaNus" now or do it later?

Reply:
not in this PR